conda/cli/main_create.py

```diff
@@ -122,4 +123,7 @@ def execute(args: Namespace, parser: ArgumentParser) -> int:
             dry_run=False,
         )
 
-    return install(args, parser, "create")
+    install(args, parser, "create")
```

Comment:
Removing this return statement won't do anything bad. The `install` function does not return any return codes. Just `None` if successful.

Reply:
The return type in the signature needs updating, or we do need to return `1` in case of errors, I think. Better double check because this is the type of oversight that ends up hitting CLI usage with unreported errors.